### ISSUE TITLE
Error: transaction rejected by network

### ISSUE THREAD

**Issue by eikira** [BTCPrivate/electrum-btcp#44]
Let me say this first. I am just a bit experienced in encoding keys in varios formats etc.
That is the situation:
I had an Electrum Wallet. It was SegWit.
I could import the seed into the BTCP Electrum Wallet. And it showed me the coins i had at the hard fork split point.

Now i wanted to send it to a coinomi but got an Error. Then i just wanted to send it to an address in the same wallet with no coins on it. Got the same Error.

So far my understanding is, the SegWit PrivateKey are not compatible with BTCP.

So how can a guy like me make the showing coins to work sending?

Here is the Error (blured out some stuff, because i dont know what you could do with it):
![Image of Error](https://img2.picload.org/image/daoidial/error.jpg)

thx

**Comment by pjain** [comment]
Also seems related to #35

**Comment by eikira** [comment]
0 reaction so far.........

**Comment by pjain** [comment]
Yeah, I'm waiting too. #sad that none of the dev team has even acknowledged this issue

**Comment by ch4ot1c** [comment]
Ongoing discussion and review regarding spending from SegWit:
https://github.com/BTCPrivate/BitcoinPrivate/pull/155

**Comment by eikira** [comment]
to me, as a non programmer, it looks like a scam right now, nearly one month passed and developer just dont give a shit... i am disappointed and the trust is shrinking and shrinking.
